stupidns/common.go

```
@@ -0,0 +1,86 @@
+
+/* Signature for result checker functions */
+type ResultChecker func(got, want []string) bool
+
+/* Map for registering and accessing checker functions */
+var CheckerDispatchTable = map[string]ResultChecker{}
+
+/* A struct representing a test case */
+type Case struct {
+	Mqueue   [][]string        `yaml:"mqueue"`
+	Expected []string          `yaml:"expected"`
+	Fetcher  string            `yaml:fetcher`
```

golangci-lint (1.62.2)

structtag: struct field tag `yaml:fetcher` not compatible with reflect.StructTag.Get: bad syntax for struct tag value

(govet)